src/hnswbuild.c

```diff
@@ -435,20 +471,27 @@ InsertTuple(Relation index, Datum *values, bool *isnull, ItemPointer heaptid, Hn
 	/* Get datum size */
 	valueSize = VARSIZE_ANY(DatumGetPointer(value));
 
-	/* Ensure graph not flushed when inserting */
+	/* Are we in the on-disk phase? */
```

Comment:
I think the previous comment is still needed here, so maybe this could be moved to the next line.